core/src/main/java/org/mule/runtime/core/internal/profiling/tracing/export/OpenTelemetrySpanExporter.java

```diff
@@ -67,17 +80,31 @@ public <T> T visit(InternalSpanExporterVisitor<T> internalSpanExporterVisitor) {
     return internalSpanExporterVisitor.accept(this);
   }
 
-  private Context resolveParentOpentelemetrySpan(InternalSpan internalSpan) {
+  private void recordSpanExceptions(InternalSpan internalSpan) {
+    internalSpan.getErrors().forEach(this::recordSpanException);
+  }
+
+  private void recordSpanException(SpanError spanError) {
+    Attributes errorAttributes = Attributes.of(
+                                               EXCEPTION_TYPE_KEY, spanError.getError().getErrorType().toString(),
+                                               EXCEPTION_MESSAGE_KEY, spanError.getError().getDescription(),
+                                               EXCEPTION_STACK_TRACE_KEY,
+                                               InternalSpanError.getInternalSpanError(spanError).getErrorStacktrace().toString(),
+                                               EXCEPTION_ESCAPED_KEY, spanError.isEscapingSpan());
+    openTelemetrySpan.addEvent("exception", errorAttributes);
```

Comment:
add this to the constants too.
Add a TODO to this ticket: W-11610439: Track if the semconv dependency (alpha) should be added.
To remember to verify if it convenient to add these constants from the semconv project